session-replacement/src/main/java/com/amadeus/session/ResetManager.java

```diff
@@ -0,0 +1,84 @@
+package com.amadeus.session;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+import com.codahale.metrics.JmxReporter;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+
+public class ResetManager {
+  private final MetricRegistry monitoring;
+  private JmxReporter reporter; 
+  private final ErrorTracker errorTracker ;
+  
+   
+  private final Meter create_metrics ;
```

Comment:
Would either put the variable name as camelCase, or as a CONSTANT_NAME.
I would also replace "metrics" by "meter", or "metric" in singular.

Reply:
fixed